### PR TITLE
I3S-Picking - Use "u_pickSegmentation" uniform

### PR DESCRIPTION
**Pull request by dryabinin-actionengine**

### CODE REVIEW

File: examples/experimental/i3s-17-and-debug/mesh-layer/mesh-layer.js [hunk 1]
```diff
@@ -144,6 +142,13 @@ export default class MeshLayer extends SimpleMeshLayer {
       });
     }
 
+    if (segmentationMode && segmentationData) {
+      model.setUniforms({
+        // eslint-disable-next-line camelcase
+        u_pickSegmentation: true
```
Comment (ibgreen) [inline examples/experimental/i3s-17-and-debug/mesh-layer/mesh-layer.js:148]: The way this is coded, it cannot be switched back if the props change?

```suggestion
      const pickFeatures = segmentationMode && segmentationData;
      model.setUniforms({
        // eslint-disable-next-line camelcase
        u_pickSegmentation: pickFeatures
```

File: examples/experimental/i3s-17-and-debug/mesh-layer/mesh-layer.js [hunk 2]
```diff
@@ -99,6 +96,7 @@ export default class MeshLayer extends SimpleMeshLayer {
   }
 
   getModel(mesh) {
+    const {segmentationMode, segmentationData} = this.props;
```
Comment (ibgreen) [inline examples/experimental/i3s-17-and-debug/mesh-layer/mesh-layer.js:99]: Not sure that calling the prop `segmentationMode` is the most intuitive name... How about `pickFeatures`?

**Review by ibgreen — APPROVED** [review]

File: examples/experimental/i3s-17-and-debug/mesh-layer/mesh-layer.js (hunk 1 above)
Comment (dryabinin-actionengine) [reply examples/experimental/i3s-17-and-debug/mesh-layer/mesh-layer.js]: Fixed

File: examples/experimental/i3s-17-and-debug/mesh-layer/mesh-layer.js (hunk 2 above)
Comment (dryabinin-actionengine) [reply examples/experimental/i3s-17-and-debug/mesh-layer/mesh-layer.js]: renamed

**Pull request merged**